test/units/testsuite-75.sh

```diff
@@ -16,6 +22,15 @@ run() {
     "$@" |& tee "$RUN_OUT"
 }
 
+disable_ipv6() {
+    sysctl -w net.ipv6.conf.all.disable_ipv6=1
+}
+
+enable_ipv6() {
+    sysctl -w net.ipv6.conf.all.disable_ipv6=0
+    networkctl reconfigure dns0
```

Comment:
'networkctl reconfigure' waits for the interface reconfigured, but does not wait for the updated setting propagated to resolved. You may need to add a loop that IPv6 DNS server passed to resolved, though not sure it is really necessary or not.